acceptancetests/repository/bundles-lxd-profile-upgrade.yaml

```diff
@@ -1,15 +1,15 @@
 series: bionic
 applications:
   lxd-profile:
-    charm: cs:~juju-qa/bionic/lxd-profile-without-devices-2
+    charm: juju-qa-lxd-profile-without-devices
```

Comment:
we need to ensure we retain in a test bundle somewhere (if not already thought about):
- revision pinning
- deploy from cs

Reply:
i'll add it into the todo. I'll construct a test specifically for this purpose.